content/en/docs/concepts/services-networking/dns-pod-service.md

```diff
@@ -267,7 +267,12 @@ The availability of Pod DNS Config and DNS Policy "`None`"" is shown as below.
 {{% capture whatsnext %}}
 
 For guidance on administering DNS configurations, check
-[Configure DNS Service](/docs/tasks/administer-cluster/dns-custom-nameservers/)
+[Configure DNS Service](/docs/tasks/administer-cluster/dns-custom-nameservers/).
+
+The [Kubernetes DNS-Based Service Discovery](https://github.com/kubernetes/dns/blob/master/docs/specification.md)
+specification describes a common standard for different implementations of Kubernetes DNS.
```

Comment:
Possible edit:
The [Kubernetes DNS-Based Service Discovery](https://github.com/kubernetes/dns/blob/master/docs/specification.md)
specification describes a standard for implementations of Kubernetes DNS.
Read this specification if you want to understand how to interface with the service or implement your own cluster DNS service.